sdk/eventhub/eventhubs-checkpointstore-tables/src/tableCheckpointStore.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CheckpointStore, PartitionOwnership, Checkpoint } from "@azure/event-hubs";
+
+export class TableCheckpointStore implements CheckpointStore {
```

Comment:
Can you add documentation here, similar to the BlobCheckpointStore?
https://github.com/Azure/azure-sdk-for-js/blob/80bd6ad77649df4e91f7f06a793df4c5a4f2dbe1/sdk/eventhub/eventhubs-checkpointstore-blob/src/blobCheckpointStore.ts#L14-L17